srcpkgs/python3-py7zr/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'python3-py7zr'
+pkgname=python3-py7zr
+version=0.11.3
+revision=1
+wrksrc="py7zr-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-pip python3-pycryptodome python3-pyannotate"
+depends="python3-pycryptodome python3-texttable"
```

Comment:
https://github.com/miurahr/py7zr/blob/2b082f41bd83022a89cd35712840170f137dff0a/setup.cfg#L37

Looks like you've omitted several packages from `depends`.

Reply:
@ahesford I already tried adding python3-toml as well.. I still get the same error..
`hostmakedepends="python3-setuptools_scm python3-wheel python3-toml python3-pycryptodome python3-pyannotate"`..

About the stuff missing from depends, I'll add them.

Reply:
now that I've checked, most of those packages are not available in the repos as well..
```
      pyzstd>=0.14.4,<0.15.0
      ppmd-cffi>=0.3.1,<0.4.0
      bcj-cffi>=0.5.0,<0.6.0
      multivolumefile>=0.2.0,<0.3.0
```